eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=Endless OS Post-Boot Flatpak Installer
-Wants=local-fs.target
-After=local-fs.target
+Wants=ostree-remount.service
+After=ostree-remount.service
 ConditionKernelCommandLine=!eos-updater-disable
+DefaultDependencies=no
```

Comment:
If you specify `DefaultDependencies=no`, you should also specify `Conflicts=shutdown.target` since it's no longer being implicitly added. See https://github.com/endlessm/eos-boot-helper/blob/master/eos-ostree-remotes-config.service for an example. Without that the installer won't be stopped if the system starts shutting down.

I'm also pretty sure that you can wait until `local-fs.target` runs. It won't run quite as early, but then you can be sure that all local filesystems are mounted. I don't see anything else in this service that would preclude it from waiting for that. But `ostree-remount.service` will almost certainly get you everything you need.

Reply:
> If you specify DefaultDependencies=no, you should also specify Conflicts=shutdown.target since it's no longer being implicitly added. See https://github.com/endlessm/eos-boot-helper/blob/master/eos-ostree-remotes-config.service for an example. Without that the installer won't be stopped if the system starts shutting down.

Good idea - I've addressed that in #144 

> I'm also pretty sure that you can wait until local-fs.target runs. It won't run quite as early, but then you can be sure that all local filesystems are mounted. I don't see anything else in this service that would preclude it from waiting for that. But ostree-remount.service will almost certainly get you everything you need.

I found that when doing so, the full `/usr` filesystem wasn't mounted, so it wasn't able to run `/usr/libexec/eos-updater-flatpak-installer`. That may not be the case for `/usr/lib/${arch-triple)`, which is what it would be installed to if I had used the debian package.